packages/web-react/src/components/FileUploader/FileUploaderAttachment.tsx

```diff
@@ -21,6 +36,31 @@ const FileUploaderAttachment = (props: SpiritFileUploaderAttachmentProps) => {
     }
   };
 
+  if (isFileImage) {
+    const reader = new FileReader();
```

Comment:
Please, refactor this scope to the function and move it to `utils.ts` in the FileUploader directory. Eg. `image2Base64Preview(file, dimension, callback)`, for example.

Reply:
function was moved to utils file